### PR TITLE
#160577430 Setup an empty react project

### PR DESCRIPTION
**Pull request by codingedward**
#### What does this PR do?
- It sets up an empty React project.
- Initializes the default `.env` file to be used for environment variables:

#### Description of Task to be completed?
- Simply use `npx create-react-app .` on the existing repository. 
- Create different environment variable files for different environments.

#### How should this be manually tested?
- Clone the github repository `git clone https://github.com/andela/ah-shakas-frontend` on your machine.
- Change your working directory to the project folder `cd ah-shakas-frontend`
- Install the required dependencies using `yarn install`
- Run the local development server using `yarn start` and you should get the following message:
  ```
  Compiled successfully!

  You can now view ah-shakas-frontend in the browser.
  
    Local:            http://localhost:3000/
    On Your Network:  http://10.10.160.30:3000/
  
  Note that the development build is not optimized.
  To create a production build, use yarn build.
  ```

#### Any background context you want to provide?
  Create React App now supports the use of different environment variables files out of the box. For 
  more information click on this link: [here](https://medium.com/@tacomanator/environments-with-create-react-app-7b645312c09d).

  Therefore, other than creating the relevant files, nothing more needs to be done to configure their support.

#### What are the relevant pivotal tracker stories?
  [#160577430](https://www.pivotaltracker.com/story/show/160577430)

#### Screenshots (if appropriate)
  N/A

### CODE REVIEW

**Review by lukorito — COMMENTED** [review]
@codingedward great job on creating the environments. I have tested this PR and I think it is ready for a merge. I have a question, do i need to add some configurations in implementing an environment for testing?

**Comment by caveinn** [comment]
@codingedward great job the commit messages, however, are showing this as a feature when it is actually a chore.

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (Arthur236) [inline public/index.html]: Change the title here into the name of our application.

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```
Comment (Arthur236) [inline public/manifest.json]: Change the name here into the name of our application.

File: public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "name": "Create React App Sample",
```
Comment (Arthur236) [inline public/manifest.json]: Change the name here into the name of our application.

**Review by Arthur236 — CHANGES_REQUESTED** [review]
Let's have just one `.env` file instead of 3 different files for each environment. Chances are we will not need more than one

**Comment by Arthur236** [comment]
Also, any particular reason why you decided to use `yarn` instead of `npm`?

**Comment by Arthur236** [comment]
Also as @caveinn has pointed out, kindly amend your commit message

**Comment by codingedward** [comment]
@lukorito, no you do not need to configure anything for you to run the application yet.

**Comment by codingedward** [comment]
@Arthur236, although the use of `yarn` was more of a preference, I have done a bit of research and found out that yarn does have some advantages over `npm` (obtained from their [README](https://github.com/yarnpkg/yarn/blob/master/README.md)):

- ***Offline Mode***: If you've installed a package before, you can install it again without any internet connection.
- ***Deterministic***: The same dependencies will be installed the same exact way across every machine regardless of install order.

- ***Network Performance***: Yarn efficiently queues up requests and avoids request waterfalls in order to maximize network utilization.

- ***Multiple Registries***: Install any package from either npm or Bower and keep your package workflow the same.

- ***Network Resilience***: A single request failing won't cause an install to fail. Requests are retried upon failure.

- ***Flat Mode***: Resolve mismatching versions of dependencies to a single version to avoid creating duplicates.
- ***More emojis***. 🐈:smiley:

However, the two are interchangeble with the exception that `npm` cannot take advantage of  the `yarn.lock` file.

I will update the PR though to consider npm as well 👍.

**Comment by Arthur236** [comment]
😄No need for that. We can stick to yarn. I just wanted to know whether you had taken the alternative into consideration

**Comment by codingedward** [comment]
@Arthur236, cool thank you 😄👍 

I have also implemented the changes you have requested.

**Comment by Arthur236** [comment]
Cool.
One more thing I forgot. Kindly clear out the `Readme.md` file. Just let it have the name of the application as the title.

**Comment by codingedward** [comment]
@Arthur236, I have implemented the change requested 👍

**Review by Arthur236 — APPROVED** [review]

**Pull request merged**